pkg/lang/golang.go

```diff
@@ -114,6 +114,7 @@ type GoInterface struct {
 	Description   string
 	Type          string
 	PossibleTypes []GoInterfacePossibleType
+	Methods       []string
```

Comment:
Do you want to use the term `InterfaceMethods` here to match what's in the config?

Reply:
This is somewhere between being consistent, and repetitive naming (`GoInterface.InterfaceMethods` feels wrong)